src/screens/ProposalDetail.js

```diff
@@ -40,7 +53,7 @@ function ProposalDetail({
 
   const { account: connectedAccount } = useWallet()
   const chartLines = useConvictionHistory(proposal)
-
+  const { stakeToken, accountBalance } = useAppState()
```

Comment:
Ah i think in this case the `stakeToken` is already passed from `App.js`

Reply:
i think that the one passed is re requestToken